docs/development.rst

```diff
@@ -539,6 +539,99 @@ For further discussion of good and bad practices regarding commits, see:
 .. _How to Get Your Change Into the Linux Kernel: https://www.kernel.org/doc/Documentation/SubmittingPatches
 
 
+.. _`Making a change`:
+
+Making a change
+---------------
+
+To make a change, create a topic branch. You can assume that you are the only
+one using that branch, so force-pushes to that branch and rebasing that branch
+is fine.
+
+When you are ready to push your change, describe the change for users of the
+package in a change fragment file. To create a change fragment file, execute:
```

Comment:
should we define further wha change fragment file is an the link to towncrier.

Reply:
DONE. Added short explanation and link to towncrier.